python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1021,13 +1020,13 @@ def kill(
         full_cmd = self.docker_cmd + ["container", "kill"]
 
         full_cmd.add_simple_arg("--signal", format_signal_arg(signal))
-        full_cmd += containers
+        full_cmd.extend(containers)
```

Comment:
```suggestion
        full_cmd += containers
```
We can't accept diff that changes code style, unless the reason is very strong, otherwise, we'd get PRs to change code style all the time. The first person to write the line chooses the style.